### PR TITLE
RSVP

### PR DESCRIPTION
**Pull request by kzyh**
- [x] Changed test endpoint to https
- [x] Added RSVP Yes page
- [x] Added RSVP No page
- [x] Added RSVP selection apge
- [x] Added sessionstorage callback variable for after auth redirect
- [x] Supports posting/putting dependent on existing rsvp

### CODE REVIEW

File: source/components/auth/auth.js [hunk 1]
```diff
@@ -17,7 +17,8 @@ class Auth extends Component {
     .then(
       (token) => {
         sessionStorage.setItem("Authorization", 'Bearer ' + token.data.data.auth);
-        this.props.history.push("/register");
+        const callback = sessionStorage.getItem('callback');
```
Comment (redsn0w422) [inline source/components/auth/auth.js]: We should have a "default" callback in case it's not set in sessionStorage already. (Probably `/register`)

Comment (kzyh) [reply source/components/auth/auth.js]: going to /start or /register will automatically place it into sessionStorage in componentWillMount before the redirect occurs in DidMount

Comment (redsn0w422) [reply source/components/auth/auth.js]: yeah but what if someone goes to `/auth`?

Comment (kzyh) [reply source/components/auth/auth.js]: I am not a smart man

**Pull request merged**